nevergrad/optimization/optimizerlib.py

```diff
@@ -1100,19 +1100,21 @@ class MetaModel(base.Optimizer):
     """Adding a metamodel into CMA."""
 
     def __init__(self, parametrization: IntOrParameter, budget: tp.Optional[int] = None, num_workers: int = 1,
-                 multivariate_optimizer: base.ConfiguredOptimizer = CMA) -> None:
+                 multivariate_optimizer: tp.Optional[base.ConfiguredOptimizer] = None) -> None:
         super().__init__(parametrization, budget=budget, num_workers=num_workers)
-        assert budget is not None
+        if multivariate_optimizer is None:
+            multivariate_optimizer = CMA if self.dimension > 1 else OnePlusOne
         self._optim = multivariate_optimizer(self.parametrization, budget,
                                              num_workers)  # share parametrization and its rng
 
     def _internal_ask_candidate(self) -> p.Parameter:
         # We request a bit more points than what is really necessary for our dimensionality (+dimension).
+        k = int((self.dimension * (self.dimension - 1)) / 2 + 2 * self.dimension + 1)
```

Comment:
a more explicit name would be helpful, k is clear for loops but barely more

Reply:
I chose "sample_size".